ts/components/PinCreationForm.tsx

```diff
@@ -100,6 +104,20 @@ export const PinCreationForm = ({ onSubmit }: Props) => {
     [isPinConfirmationValid]
   );
 
+  const bulletList = (items: Array<string>) => (
+    <FlatList
+      data={items}
+      renderItem={({ item }) => (
+        <Body style={styles.bulletList}>{`\u2022 ${item}`}</Body>
+      )}
+    />
+  );
+
+  const insertValidPin = () => {
+    setPin("162534");
+    setPinConfirmation("162534");
```

Comment:
Maybe It'd be better to put this pattern `162534` in a shared constant (e.g. `config.ts`) 'cause we use it in many places. What do you think?